data/os/RedHat.yaml

```diff
@@ -9,7 +9,7 @@ sshd::config:
   PubkeyAuthentication: "no"
   UsePAM: "yes"
 sshd::config_subsystems:
-  sftp: "/usr/libexec/openssh/sftp-server"
+  sftp: "/usr/libexec/openssh/sftp-server -l VERBOSE"
```

Comment:
This is the only real functional change, correct?

All other changes are related to linting or the pdk update.

Reply:
Correct yeah, that line is the only real change here.